db/seed_maker.rb

```diff
@@ -382,6 +382,8 @@ def load_hmis_data
       builder.seed_record_form_definitions
       # Load ALL the latest assessment definition froms JSON files.
       builder.seed_assessment_form_definitions
+      # Load admin forms (not configurable)
+      builder.seed_static_forms
       # In development, create the initial instances for occurrence-point collection.
       builder.create_default_occurrence_point_instances! if Rails.env.development?
     end
```

Comment:
Alternatively we could replace this block with 
```
Rails.application.load_tasks
Rake::Task['driver:hmis:seed_definitions'].invoke
```

That seems preferable to me, only 1 thing to maintain, but the load_tasks seems expensive and I'm not sure if it's ok to do it here

Reply:
I think I prefer this.  Not sure why we'd invoke a rake task here if we have the single method that accomplishes what we need.

Reply:
The logic is duplicated between this file and the rake task (lines 379-389)